cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -777,37 +776,43 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             value: account.outputs.endpoint
           }
           {{- end}}
-          {{- if .AzureEventHubs }}
+          {{- if and .AzureEventHubs (not .AzureEventHubs.UseKafka) }}
           {
             name: 'SPRING_CLOUD_AZURE_EVENTHUBS_NAMESPACE'
             value: eventHubNamespace.outputs.name
           }
           {{- end}}
-          {{- if (and .AzureEventHubs .AzureEventHubs.AuthUsingManagedIdentity) }}
+          {{- if and .AzureEventHubs .AzureEventHubs.UseKafka }}
           {
-            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTIONSTRING'
-            value: ''
+            name: 'spring.cloud.stream.kafka.binder.brokers'
+            value: '${eventHubNamespace.outputs.name}.servicebus.windows.net:9093'
           }
+          {{- end}}
+          {{- if and .AzureEventHubs .AzureEventHubs.AuthUsingManagedIdentity }}
```

Comment:
It's better to add `()` for `and` operator.

Reply:
Updated.